File: SubSequence/Subsequence.py
class SubSequence:
    def printsubsequence(self,pstr,unstr):
        if(len(unstr) == 0):
            print(pstr)
            return

        self.printsubsequence(pstr+unstr[0], unstr[1:])
        self.printsubsequence(pstr, unstr[1:])
        return

    
obj = SubSequence()
obj.printsubsequence("","abc")

File: SubSequence/SubSequence.java
public class SubSequence {

    void sequence(String Processed_String, String UnProcessed_String){
        if(UnProcessed_String.isEmpty()){
            System.out.println(Processed_String);
            return;
        }
        this.sequence( Processed_String + UnProcessed_String.charAt(0), UnProcessed_String.substring(1));
        this.sequence(Processed_String , UnProcessed_String.substring(1));

    }
    public static void main(String[] args) {
        SubSequence obj = new SubSequence();
        obj.sequence("", "123");
        
    }
}
